packages/workbox-background-sync/src/lib/request-queue.js

```diff
@@ -96,6 +96,7 @@ class RequestQueue {
    * @private
    */
   async push({request}) {
+    await this._initializationPromise;
```

Comment:
@gauntface quick question: method
`get queue() {...}` is only used at one place, do you think making this async itself will be useful in longer term?